docs/build_and_load.md

```diff
@@ -442,6 +498,38 @@ or review the following code blocks:
 
 [//]: # (| ARM       | 34.2             | 18:08:57  |)
 
+### Annotate Fastas
```

Comment:
This section is actually written out in further detail under the "Compress FASTA files" section.

Reply:
You are changing this, correct?

Reply:
Ie, deleting what I have, moving your stuff up where it makes sense? Or I can add it - let me know.

Reply:
This has been moved and corrected collaboratively, so I will mark this as resolved